bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -662,12 +678,23 @@ attributes from the config_cache.xml file (with keys converted to upper-case).
 <use_init_interp  use_cndv=".false." use_fates=".false." sim_year="2013" lnd_tuning_mode="clm5_0_cam6.0"
                   maxpft="17"        use_cn=".false."    use_crop=".false." hgrid="ne0np4CONUS.ne30x8"      >.true.</use_init_interp>
 <!-- For an inexact match use either low resolution or high resolution match for SP or BGC mode CLM5.0 -->
-<use_init_interp  use_cndv=".false." use_fates=".false." sim_year="2000"  phys="clm5_0"                     >.true.</use_init_interp>
-<use_init_interp  use_cndv=".false." use_fates=".false." sim_year="2000"  phys="clm5_1"                     >.true.</use_init_interp>
+<use_init_interp use_cndv=".false." use_fates=".false." sim_year="2000" phys="clm5_0" lnd_tuning_mode="clm5_0_GSWP3v1" >.true.</use_init_interp>
+<use_init_interp use_cndv=".false." use_fates=".false." sim_year="2000" phys="clm5_0" lnd_tuning_mode="clm5_0_CRUv7" >.true.</use_init_interp>
+<use_init_interp use_cndv=".false." use_fates=".false." sim_year="2000" phys="clm5_0" lnd_tuning_mode="clm5_0_cam6.0" >.true.</use_init_interp>
+<use_init_interp use_cndv=".false." use_fates=".false." sim_year="2000" phys="clm5_1" lnd_tuning_mode="clm5_1_GSWP3v1" >.true.</use_init_interp>
+<use_init_interp use_cndv=".false." use_fates=".false." sim_year="2000" phys="clm5_1" lnd_tuning_mode="clm5_1_CRUv7" >.true.</use_init_interp>
+<use_init_interp use_cndv=".false." use_fates=".false." sim_year="2000" phys="clm5_1" lnd_tuning_mode="clm5_1_cam6.0" >.true.</use_init_interp>
 <!-- For an inexact match use either low resolution or high resolution match for SP or BGC mode CLM4.5 -->
 <use_init_interp  use_cndv=".false." use_fates=".false." sim_year="2000"  lnd_tuning_mode="clm4_5_GSWP3v1"  >.true.</use_init_interp>
 <use_init_interp  use_cndv=".false." use_fates=".false." sim_year="2000"  lnd_tuning_mode="clm4_5_CRUv7"    >.true.</use_init_interp>
 <use_init_interp  use_cndv=".false." use_fates=".false." sim_year="1979"  lnd_tuning_mode="clm4_5_cam6.0"   >.true.</use_init_interp>
+<!-- Let CAM4 and CAM5 lnd_tuning mode be FALSE (so do a coldstart) -->
+<use_init_interp lnd_tuning_mode="clm4_5_cam5.0" >.false.</use_init_interp>
+<use_init_interp lnd_tuning_mode="clm4_5_cam4.0" >.false.</use_init_interp>
+<use_init_interp lnd_tuning_mode="clm5_0_cam5.0" >.false.</use_init_interp>
+<use_init_interp lnd_tuning_mode="clm5_0_cam4.0" >.false.</use_init_interp>
+<use_init_interp lnd_tuning_mode="clm5_1_cam5.0" >.false.</use_init_interp>
+<use_init_interp lnd_tuning_mode="clm5_1_cam4.0" >.false.</use_init_interp>
```

Comment:
Worth checking whether cam4 will still be supported when we release this.

Reply:
I asked in CSEG about this, and they are even going to port CAM4 into one of the newer CAM structures (was it CAM-SIMA ?). So it will be securely supported moving forward so we will need to support it as well. I think they did say that CAM3 is going to be removed. This is tangentially also good for SLIM that CAM4 and CAM5 are going to be supported.